core/types/calls.cc

```diff
@@ -168,7 +168,7 @@ unique_ptr<Error> matchArgType(Context ctx, TypeConstraint &constr, Loc callLoc,
         auto withoutNil = Types::approximateSubtract(ctx, argTpe.type, Types::nilClass());
         if (!withoutNil->isBottom() && Types::isSubTypeUnderConstraint(ctx, constr, withoutNil, expectedType)) {
             if (loc.exists()) {
-                e.replaceWith(loc, "T.must({})", loc.source(ctx));
+                e.replaceWith("Add `T.must`", loc, "T.must({})", loc.source(ctx));
```

Comment:
This should probably be worded as "Wrap in `T.must`" :)

Reply:
Also, FWIW VS Code renders the backticks literally in the correction options, I'm not sure if that matters but I figured I'd mention it.

Reply:
Good idea!

Reply:
And yes, unfortunately, there's no Markdown rendering but I preferred this over no distinction.